scripts/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:12 AS cuttlefish-cloud-orchestration
+
+RUN set -x
+RUN apt update
+RUN apt install -y --no-install-recommends \
+    ca-certificates \
+    docker.io \
+    golang \
+    sudo
+RUN update-ca-certificates
+
+COPY . /root/cloud-android-orchestration
+WORKDIR /root/cloud-android-orchestration
+
+RUN go build ./cmd/cloud_orchestrator
```

Comment:
maybe we want multi-staging docker file as well as docker image for HO because golang isn't required in the runtime, i guess

Reply:
Not only for `golang`, but also not keeping all source code inside `cloud-android-orchestration` repository. If anyone here like this approach, I'll be working on optimization things to provide better image.

Reply:
Applied multi-staging docker file for cloud orchestrator. Closing this comment.